app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/TimeUtils.scala

```diff
@@ -88,7 +88,8 @@ object TimeUtils {
         key: String,
         data: Map[String, String]
       ): Either[Seq[FormError], LocalDate] = {
-        val result = for {
+        val minimumDate = LocalDate.of(1900, 1, 1)
```

Comment:
we could move this out of the `def` to avoid creating a new `LocalDate` which is the same value every time this function is called